src/components/OttList.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import { Box } from '@mui/material';
+import OttItem from './OttItem';
+import { useState } from 'react';
+
+const OttList = ({ ottServices }) => {
+  const [selectedId, setSelectedId] = useState(0);
+  const handleSelectOtt = (ottId) => {
+    setSelectedId(ottId);
+  };
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexWrap: 'wrap',
+        p: 1,
+        m: 1,
+        bgcolor: 'background.paper',
+        maxWidth: 300,
+      }}
+    >
+      {ottServices.map(({ ottId, ottName }) => (
+        <OttItem
+          key={ottId}
+          ottId={ottId}
+          ottName={ottName}
+          selectedId={selectedId}
```

Comment:
```suggestion
          selected = { ottId === selectedId }
```
선택된 OTTService의 id가 OttItem 컴포넌트에서 따로 쓰이는 것이 아니라면 선택되었는지 알려주는 boolean 값만 내려줘도 괜찮을 것 같다 생각이 드는데 낸시 생각은 어떠실까요??

Reply:
ottItem 컴포넌트에서 검사를 하기보다는 boolean값만 전달하는 것이 깔끔하겠네요👍